.github/labeler.yml

```diff
@@ -0,0 +1,38 @@
+# A bot for automatically labelling pull requests 
+# See https://github.com/actions/labeler
+
+dataframe:
+  - changed-files:
+    - any-glob-to-any-file:
+      - crates/nu_plugin_polars/**
+
+std-library:
+  - changed-files:
+    - any-glob-to-any-file:
+      - crates/nu-std/**
+
+ci:
+  - changed-files:
+    - any-glob-to-any-file:
+      - .github/workflows/**
+
+
+LSP:
+  - changed-files:
+    - any-glob-to-any-file:
+      - crates/nu-lsp/**
+
+parser:
+  - changed-files:
+    - any-glob-to-any-file:
+      - crates/nu-parser/**
+
+pr:plugins:
+  - changed-files:
+    - any-glob-to-any-file:
+      - crates/nu-plugin/**
```

Comment:
can we also add `crates/nu_plugin_polars/**` here so that it gets two labels? or does it not allow two?

Reply:
Yes! you can add all the folders under nu_plugin_* and it the case of nu_plugin_polars should tag `dataframe` and `pr:plugins`

I don't know if `pr:plugins` is intended to label _just_ the plugin API or also all the existing plugins

Reply:
I've been labeling all polars changes with pr:plugins for months now.

Reply:
Done!